### PR TITLE
Update references to use dependencies from ConanCenter

### PR DESCRIPTION
**Pull request by danimtb**
Updated references and commands to remove user/channel.

- Should we keep `user/testing` for user-created packages?

⚠️ Slides are not updated with these changes yet!

### CODE REVIEW

File: catchup.sh [hunk 1]
```diff
@@ -74,18 +74,18 @@ create() {
    echo "performing Exercise 7 (Create a Conan Package)"
    cd create
    conan new hello/0.1
-   conan create . user/testing
```
Comment (memsharded) [inline catchup.sh]: Yes, lets keep user created packages with user/name

Comment (danimtb) [reply catchup.sh]: Done!

File: catchup.sh [hunk 2]
```diff
@@ -249,25 +249,25 @@ lockfiles(){
     # This will use the 
     conan create chat user/testing
     # the chat package will NOT use 0.2 it is locked to 0.1
-    conan create chat user/testing --lockfile
+    conan create chat --lockfile
```
Comment (memsharded) [inline catchup.sh]: This is my package, needs to use user/testing

File: consumer_cmake_find/conanfile.txt [hunk 3]
```diff
@@ -1,10 +1,10 @@
 [requires]
-boost/1.67.0@conan/stable
-Poco/1.9.0@pocoproject/stable
+boost/1.72.0
+poco/1.9.4
 
 [generators]
-cmake_find_pakcage
```
Comment (memsharded) [inline consumer_cmake_find/conanfile.txt]: This bug was on purpose. To keep students alert, learn to interpret Conan error messages, etc.

**Pull request merged**